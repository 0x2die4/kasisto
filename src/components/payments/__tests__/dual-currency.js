/* eslint-env jest */

import React from 'react'
import DualCurrency from '../dual-currency'
import renderer from 'react-test-renderer'

describe('DualCurrency', () => {
  it('renders fiat and money', () => {
    const component = renderer.create(
      <DualCurrency />
    )

    let tree = component.toJSON()

    // expect(Payment.on).toBeCalledWith('ready')

    expect(tree).toMatchSnapshot()
  })
})
