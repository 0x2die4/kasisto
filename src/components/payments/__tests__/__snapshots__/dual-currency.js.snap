// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DualCurrency renders fiat and money 1`] = `
<div
  className="c-dual-currency "
>
  <span
    className="c-currency c-currency--primary"
  >
    <span
      className="c-currency__value"
    >
      NaN
    </span>
    <span
      className="c-currency__tail"
    >
      .00
    </span>
    <span
      className="c-currency__currency"
    >
      XMR
    </span>
  </span>
  <span
    className="c-currency c-currency--secondary"
  >
    <span
      className="c-currency__value"
    >
      NaN
    </span>
    <span
      className="c-currency__tail"
    >
      .00
    </span>
    <span
      className="c-currency__currency"
    >
      XMR
    </span>
  </span>
</div>
`;
